app/src/main/java/com/duckduckgo/app/di/AppComponent.kt

```diff
@@ -31,22 +31,26 @@ import com.duckduckgo.app.onboarding.di.WelcomePageModule
 import com.duckduckgo.app.surrogates.di.ResourceSurrogateModule
 import com.duckduckgo.app.trackerdetection.di.TrackerDetectionModule
 import com.duckduckgo.app.usage.di.AppUsageModule
+import com.duckduckgo.di.scopes.AppObjectGraph
 import com.duckduckgo.widget.SearchWidget
+import com.squareup.anvil.annotations.MergeComponent
 import dagger.BindsInstance
 import dagger.Component
 import dagger.android.AndroidInjector
 import dagger.android.support.AndroidSupportInjectionModule
 import kotlinx.coroutines.CoroutineScope
+import retrofit2.Retrofit
+import javax.inject.Named
 import javax.inject.Singleton
 
 @Singleton
-@Component(
+@MergeComponent(
+    scope = AppObjectGraph::class,
```

Comment:
`@MergeComponent` is the anvil-equivalent to dagger `@Component`. This annotation is what anvil uses to generate the dagger components under the hood.
The `scope` is not be confused to eg. `ActivityScope` or `Singleton` scope-annotations. It simply is a marker class that identifies the dagger component, so that we can later provide modules/dependencies without needing to get access to the dagger component type